### ISSUE TITLE
Text Search- entrega 5 - user id

### ISSUE THREAD

**Issue by Tomas-Ira** [IIC2413/Syllabus-2020-1#425]
Tengo una duda respecto a los criterios a la hora de realizar el text-search. En la entrega aterior, el 4to criterio era indicar el id del usuario que envía el mensaje. Para esta entrega, como el usuario está logeado a una cuenta, no debo pedir este atributo y debo asumir que solo utilizo el id del usuario loggeado o no?

**Comment by diegomendieta** [comment]
Misma duda.

**Comment by nivek0o0** [comment]
No, el criterio de userId del text search (4ta funcionalidad pedida en el enunciado) no tiene nada que ver con el usuario loggeado. Debe ser posible pasar cualquier ID.

**Comment by nivek0o0** [comment]
Para los mensajes enviados (2da funcionalidad del enunciado) puedes utilizar text search con el criterio userId como el logeado, aunque también hay otras formas, como obtener todos los mensajes y filtrar con Python, o como pedir el usuario loggeado y obtener sus mensajes que deberían venir anidados.
